### PR TITLE
Keep Dashboard Legacy Warning

### PR DESCRIPTION
**Pull request by r-czajkowski**
Ref: https://github.com/threshold-network/token-dashboard/issues/488

This PR adds a warning explaining that the Keep dashboard is a legacy dashboard. Here we display a Legacy dashboard warning modal(only once)- when the modal is dismissed, it will not be shown again. Apart from that, we display a permanent ribbon banner. Here we also remove unsupported features:
- tBTC token migration portal page,
- liquidity page,
- remove the possibility to deposit in the coverage pools since there are no longer rewards. Users can only withdraw tokens from pool.

### CODE REVIEW

**Comment by erdogan** [comment]
@r-czajkowski can we make the following changes to the modal content (I currently don't have edit access to Sasha's figma files so noting changes here) cc @sashatanase 

## Keep Network Dashboard is in legacy mode
Only the following features are supported in legacy mode: 
- Stake undelegation
- Token grant withdrawal
- Stake upgrade for Threshold Network
- tBTC v1 token upgrade

Visit [Threshold Dashboard](url) for the tBTC v2 dApp and staking in Threshold Network.

**Review by michalsmiarowski — COMMENTED** [review]
Overall looks great! Let's fix those issues that @erdogan mentioned and I think we can merge

File: solidity-v1/dashboard/src/components/modal/LegacyDashboardModal.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          <Button className="btn btn-primary btn-lg mr-2" onClick={_onClose}>
+            stay on page
+          </Button>
+          <a
+            href={LINK.thresholdDapp}
+            rel="noopener noreferrer"
+            target="_blank"
+            className="no-arrow text-grey-70"
+          >
+            Go to Threshold dashboard
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity-v1/dashboard/src/components/modal/LegacyDashboardModal.jsx:93]: There is no arrow in Figma, so I added `no-arrow` CSS class. I think we can leave it as is.

**Review by michalsmiarowski — APPROVED** [review]
LGTM

**Comment by erdogan** [comment]
In <List.item>Token grant withdawal</List.item>, we are missing an **r** in withd**r**awal. Otherwise looks good.

**Pull request merged**